build-utils/integration-docs-fetch-plugin.ts

```diff
@@ -68,8 +72,13 @@ class IntegrationDocsFetchPlugin {
   fetch: Parameters<webpack.Compiler['hooks']['beforeRun']['tapAsync']>[1] = (
     _compilation,
     callback
-  ) =>
-    https
+  ) => {
+    let httpClient = https;
+    if (url.parse(PLATFORMS_URL).protocol === 'http:') {
+      // @ts-ignore
+      httpClient = http;
+    }
+    return httpClient
```

Comment:
Is this so that it works on localhost?

Reply:
Nit, but I would prefer returns here

```
    if (url.parse(PLATFORMS_URL).protocol === 'http:') {
       return http
    }
    return https
```

Reply:
yea, and frankly i found this fetch-plugin didn't have an effect for me, but i've aligned it to what i've done on the python side just to make sure its covered.